news.html

```diff
@@ -8,13 +8,21 @@
 
     <div class="container community-wrapper">
 
-        <div class="col-sm-8 wrapper">
+        <div class="col-sm-12 wrapper">
```

Comment:
It is intentional. It's for general readability and UX, as there is very little text on the page. Please keep as is.